<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #282828</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="randomLayout"/>
  </widget>
  <widget class="QPushButton" name="generate_btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                            color: rgb(200, 200, 200);
                            border-style: 2px solid rgb(200, 200, 200);
                            border-radius: 7px;
                            background-color: #383838;
                            }

                            QPushButton:hover{
                            background-color: #303030;
                            }
                        </string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>21</y>
     <width>298</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(180,180,180);
                            font-size: 15px;
                            background-color: transparent;
                        </string>
   </property>
   <property name="text">
    <string>Choose type of random based on distribution:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="distBox">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>49</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
                            color: rgb(200, 200, 200);
                            border-style: none;
                            border-radius: 7px;
                            background-color: #383838;
                            padding-left: 13px;
                            }

                            QComboBox::drop-down
                            {
                            border: 0px; /* This seems to replace the whole arrow of the combo box */
                            }
                            QComboBox::down-arrow {
                            image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
                            width: 9px;
                            height: 9px;
                            padding-right: 10px;
                            }
                        </string>
   </property>
  </widget>
  <widget class="QLabel" name="notiLabel">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>219</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(180,180,180);

                            background-color: transparent;
                        </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
